### DISCUSSION TITLE
invalid version: unknown revision 000000000000 for github.com/docker/distribution

### DISCUSSION THREAD

**Discussion by nagou** [oras-project/oras#235] [Q&A] [answered]
How can I fix following?

go get: helm.sh/helm/v3@v3.5.2 requires
        github.com/deislabs/oras@v0.10.0 requires
        github.com/docker/distribution@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000

**Comment by deitch** [comment, marked as answer]
Already addressed in https://github.com/deislabs/oras/issues/229

**Reply by nagou** [reply]
cool! Thanks for quick reply! Appreciate
